fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -300,6 +300,105 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 	ACTOR Future<Void> testSpecialKeySpaceErrors(Database cx_, SpecialKeySpaceCorrectnessWorkload* self) {
 		Database cx = cx_->clone();
 		state Reference<ReadYourWritesTransaction> tx = makeReference<ReadYourWritesTransaction>(cx);
+		state Reference<ReadYourWritesTransaction> tenantTx = makeReference<ReadYourWritesTransaction>(cx);
+		state Reference<ReadYourWritesTransaction> tenantTx2 = makeReference<ReadYourWritesTransaction>(cx);
```

Comment:
For the first few functions where we are explicitly checking for tenant failures, you will need to create a transaction that specifies a tenant.

For the subsequent checks that conflict ranges are reported properly, those should use the default tenant and you could probably use transactions named `tx` and `tx2` since they may or may not have tenants.